.github/workflows/macos.yml

```diff
@@ -45,7 +42,7 @@ jobs:
       run: |
         mkdir upload
         mv ${{github.workspace}}/build/shadps4 upload
-        cp /usr/local/lib/libMoltenVK.dylib upload
+        cp /opt/homebrew/opt/molten-vk/lib/libMoltenVK.dylib upload
```

Comment:
This shouldn't be changed to `opt/homebrew`. 
x86 binaries are kept in `usr/local` and arm binaries are kept in `opt/homebrew`

Reply:
molten-vk didn't exist in usr/local